elastic.js/elastic.js.d.ts

```diff
@@ -139,7 +139,7 @@ declare module elasticjs {
      Retrieves the internal agg object. This is typically used by
      internal API functions so use with caution.
      */
-    toJSON(): String;
+    toJSON(): Object;
```

Comment:
Yes, all this kind of places has been changed so there are "any" instead of "Object" now. Please check.